lib/plug/static.ex

```diff
@@ -7,12 +7,12 @@ defmodule Plug.Static do
     * `:at` - the request path to reach for static assets.
       It must be a string.
 
-    * `:from` - the filesystem path to read static assets from.
-      It must be a string, containing a file system path, or an
-      atom representing the application name where assets will
-      be served from the priv/static, or a tuple containing the
-      application name and the directory to serve assets besides
-      priv/static.
+    * `:from` - the file system path to read static assets from.
```

Comment:
I think I explained myself bad once again 😄 I meant that we either go with one or the other, not both :) and since there's no agreement in the Elixir docs on which to choose from "filesystem" or "file system", we could have left this unchanged :) But the change is fine anyways of course, just wanted to clarify things a bit 😄 Sorry for the noise!